### PR TITLE
Document page styling

### PR DESCRIPTION
**Pull request by whuang7000**
# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
Styled document upload page
Screen Shot 2022-02-28 at 1 00 15 AM

Screen Shot 2022-02-28 at 1 00 33 AM

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [X] Page text
 - [X] Modal
 - [ ] Upload tutorial (added todo for new mem sprint task) 
 - [ ] Style buttons in the modal

## How can the reviewer test your code? 👩‍💻
Go to upload page

## Any bugs you encountered or still having trouble with? 🐛
_describe how you solved them such that someone can follow your explanation_ 

## Resources 📔
_link online resources and related PRs_ 

🧜‍♀️ cc: @shannonbonet @phoebeli23

### CODE REVIEW

**Comment by bry-gavino** [comment]
thank you for the PR will!! I'm liking how it's looking so far, but I'm gonna go over this with Jared before merging. thanks again!

File: src/screens/Upload/CameraScreen.tsx [hunk 1]
```diff
@@ -3,27 +3,42 @@ import * as ImagePicker from 'expo-image-picker';
 import firebase from 'firebase/app';
 import React, { useEffect, useState } from 'react';
 import {
-  Button,
   StatusBar,
   LogBox,
```
Comment (j-keating0002) [inline src/screens/Upload/CameraScreen.tsx:7]: i allowed access to all photos + camera but when trying to takes photos received error: missing camera or camera roll permissions
image
possible fix: https://github.com/expo/expo/issues/4826 using expo-camera package?

**Review by j-keating0002 — CHANGES_REQUESTED** [review]
- camera permissions error
- fixed some styling errors

**Comment by bry-gavino** [comment]
> bugs:
> 
> * camera permissions error
> * selecting new photos when adding a page removes all of the previous uploaded photos
> 
> fixed:
> 
> * fixed some styling errors

Hey Jared, is this ready for merging?

**Review by bry-gavino — APPROVED** [review]
Hey Will! Thanks for the great implementation and styling! Can't wait to merge this and have some new mems work on those tasks you mentioned!

**Pull request merged**